http/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "http"
+version = "0.1.0"
+authors = ["Joonas Koivunen <joonas@equilibrium.co>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```

Comment:
I like ordering deps alphabetically as it helps readability once they're 10-20 dependencies

Reply:
Reordered